src/azure-cli/azure/cli/command_modules/security/_params.py

```diff
@@ -73,6 +75,15 @@ def load_arguments(self, _):
                 validator=validate_auto_provisioning_toggle,
                 arg_type=auto_provisioning_auto_provision_arg_type)
 
+    for scope in ['atp storage show']:
+        with self.argument_context('security {}'.format(scope)) as c:
+            c.argument('storage_account_name', help='Name of an existing storage account.')
+
+    for scope in ['atp storage update']:
+        with self.argument_context('security {}'.format(scope)) as c:
+            c.argument('storage_account_name', help='Name of an existing storage account.')
```

Comment:
storage account name can be abstracted using CLIArgumentType and reuse it for different commands.